zktest/pom.xml

```diff
@@ -425,9 +425,9 @@
 				</configuration>
 			</plugin>
             <plugin>
-                <groupId>org.mortbay.jetty</groupId>
+                <groupId>org.eclipse.jetty</groupId>
                 <artifactId>jetty-maven-plugin</artifactId>
-                <version>7.6.8.v20121106</version>
+                <version>9.4.2.v20170220</version>
```

Comment:
Jetty 9.4 is Java 8+ only.
Check Jenkins' installed Java version to ensure it will not be broken after merging.